### ISSUE TITLE
Readme incorrectly states range for `cpu.maxLoad` and `cpu.incrementBy`

### ISSUE THREAD

**Issue by fbuchmeier-abi** [woehrl01/pod-pacemaker#35]
Hi

I've noticed that according to the [Readme](https://github.com/woehrl01/pod-pacemaker/blob/main/README.md?plain=1#L77) the value range for `cpu.maxLoad` and `cpu.incrementBy` should be 0.0 - 1.0 for 0% and 100% load respectively.

>   - `maxLoad`:  [...] e.g. `0.8` means 80% of CPU usage.
>   - `incrementBy`: [...] e.g. `0.1` means 10% of CPU usage.

As far as I can see however, both values are compared against the cpu percentage, so a range of 0-100 (%) would be correct:

https://github.com/woehrl01/pod-pacemaker/blob/main/pkg/throttler/cpuload.go#L31C52-L31C63

```
load, err := cpu.Percent(5*time.Second, perCpu)
```

This is also the case on my test cluster, where setting `cpu.maxLoad` means that no new pods can be scheduled and lock requests time out (as no node has less than 8% CPU usage even when idling).

Best regards,
Florian

PS: Awesome project for a really annoying problem.

**Comment by woehrl01** [comment]
Thank you @fbuchmeier-abi

You are absolutely right. What do you think? Should we fix the readme, or should we keep the fraction and change the calculation?

**Comment by fbuchmeier-abi** [comment]
I've fixed the Readme in my Pull Request. However I need to wait for my company to approve it before I can push it to github and I don't think this will happen until next Tuesday :)

I am also working on a small test setup to validate my pacemaker configuration. It uses a simple deployment that performs a fixed amount of work during start up that can be scaled based on the current test scenario.

If  you are interested I can provide you with the details in a separate issue once I have refined it a little bit more (probably at the end of next week). So far it is looking  great and I can reduce the time it takes for my test setup to spin up by 70% compared to not using the pacemaker.
